packages/runtime/client/rest/src/index.ts

```diff
@@ -253,7 +265,9 @@ export function mkRestClient(baseURL: string): RestAPI {
         )
       );
     },
-
+    createContractSources(mainId, bundle) {
```

Comment:
Why not an Abstraction called `ContractSource` with a main and a bundle ?

Reply:
Early abstraction. Maybe Bundle should have the main as well. once I implement the other methods of contract source this will become clearer